src/datachain/lib/dc.py

```diff
@@ -577,6 +572,43 @@ def datasets(
             **{object_name: datasets},  # type: ignore[arg-type]
         )
 
+    @classmethod
+    def listings(
+        cls,
+        session: Optional[Session] = None,
+        in_memory: bool = False,
+        object_name: str = "listing",
+        **kwargs,
+    ) -> "DataChain":
+        """Generate chain with list of cached listing.
```

Comment:
it would be great to explain a bit what the listing is - just a few words

Reply:
Added one sentence